ntoskrnl/mm/kasan.c

```diff
@@ -0,0 +1,72 @@
+/*
+ * PROJECT:     ReactOS Kernel
+ * LICENSE:     GPL-2.0+ (https://spdx.org/licenses/GPL-2.0+)
+ * PURPOSE:     Kernel address sanitizer. Manage shadow memory
+ *              for kernel address sanitization.
+ * COPYRIGHT:   Copyright 2022 Nick DiMeglio
+ */
+
+/* INCLUDES ******************************************************************/
+
+#include <ntoskrnl.h>
+#define NDEBUG
+#include <debug.h>
+
+#define MODULE_INVOLVED_IN_ARM3
+#include "ARM3/miarm.h"
+
+/* GLOBALS *******************************************************************/
+
+#define MM_KASAN_SHADOW_MEMORY 0x90000000
+#define MM_KASAN_SHADOW_SIZE 0x10000000
```

Comment:
Something like
```suggestion
#define MM_KASAN_SHADOW_SIZE (MM_KASAN_SHADOW_MEMORY_END - MM_KASAN_SHADOW_MEMORY)
```
Moreover, it's only used once, so this new define seems overkill.